docs/7.x/config.md

```diff
@@ -404,6 +404,8 @@ spec:
     - {% raw %}{claim: "roles", value: "gravitational/admins", roles: ["@teleadmin"]}{% endraw %}
 ```
 
+!!! note For Auth0 the OIDC Conformant setting should be off in Advanced Setting -> OAuth or Claims will not populate properly 
```

Comment:
Nit: Could you add a dot at the end of the sentence please?

Also, shouldn't the note be formatted like this:

```
!!! note:
  <note text>
```

Disregard if your way is displayed properly as well.

Reply:
think it would still work but modified to make it uniform with other notes in the doc